### PR TITLE
add merklechain 

### PR DESCRIPTION
**Pull request by ankyjais**

### CODE REVIEW

File: _data/chains/eip155-1909.json
Comment (macvts) [inline _data/chains/eip155-1909.json:1]: remove / at the end of infoURL and URL
 @ankyjais

Comment (macvts) [reply _data/chains/eip155-1909.json:1]: is merklescan's ipfs sizing correct?

Comment (tilemanrenovations) [reply _data/chains/eip155-1909.json:1]: No it is not.

**Comment by ankyjais** [comment]
Kindly merge the Merklechain in your chain list

**Review by tilemanrenovations — APPROVED** [review]

**Review by ligi — APPROVED** [review]

**Pull request merged**